src/app/static/src/app/store/modelCalibrate/modelCalibrate.ts

```diff
@@ -0,0 +1,39 @@
+import {Module} from "vuex";
+import {RootState} from "../../root";
+import {DynamicFormData, DynamicFormMeta} from "@reside-ic/vue-dynamic-form";
+import {mutations} from "./mutations";
+import {localStorageManager} from "../../localStorageManager";
+import {actions} from "./actions";
+
+export interface ModelCalibrateState {
+    optionsFormMeta: DynamicFormMeta
+    options: DynamicFormData
+    fetching: boolean
+    complete: boolean
+}
+
+export const initialModelCalibrateState = (): ModelCalibrateState => {
+    return {
+        optionsFormMeta: {controlSections: []},
+        options: {},
+        fetching: false,
+        complete: false
+    }
+};
+
+export const modelCalibrateGetters = {
+    complete: (state: ModelCalibrateState) => {
+        return state.complete;
```

Comment:
Why have a getter here when can just access the state prop directly?

Reply:
I think I put this in when I wasn't sure what the API schema would be and thought this might end up being a more complex check on the returned data, but that isn't the case, so yes doesn't need to be a getter.